books/free-programming-books-pt_BR.md

```diff
@@ -387,7 +387,7 @@
 
 ### Pascal
 
-* [Algoritmos e Estruturas de Dados I](https://www.inf.ufpr.br/marcos/livro_alg1/livro_alg1.pdf) - Marcos Alexandre Castilho, Fabiano Silva, Daniel Weingaertner (PDF)
+* [Algoritmos e Estruturas de Dados 1](https://www.inf.ufpr.br/marcos/livro_alg1/livro_alg1.pdf) - Marcos Castilho, Fabiano Silva, Daniel Weingaertner (PDF) (CC BY-NC-ND)
```

Comment:
Looks like this is a duplicate listing! we only allow one entry per resource. Thanks for noticing!! (please remove it)

Reply:
Sure! I'll change right away.